src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -0,0 +1,493 @@
+from blockkit import Context, Input, MarkdownText, Modal, Section, UsersSelect
+
+from dispatch.case import flows as case_flows
+from dispatch.case import service as case_service
+from dispatch.case.enums import CaseStatus
+from dispatch.case.models import CaseCreate, CaseUpdate
+from dispatch.incident import flows as incident_flows
+from dispatch.plugins.dispatch_slack.bolt import app
+from dispatch.plugins.dispatch_slack.case.enums import (
+    CaseEscalateActions,
+    CaseNotificationActions,
+    CaseReportActions,
+    CaseResolveActions,
+    CaseShortcutCallbacks,
+)
+from dispatch.plugins.dispatch_slack.case.messages import (
+    create_case_message,
+)
+from dispatch.plugins.dispatch_slack.fields import (
+    DefaultBlockIds,
+    case_priority_select,
+    case_status_select,
+    case_type_select,
+    description_input,
+    incident_priority_select,
+    incident_type_select,
+    project_select,
+    resolution_input,
+    title_input,
+)
+from dispatch.plugins.dispatch_slack.middleware import (
+    action_context_middleware,
+    button_context_middleware,
+    db_middleware,
+    modal_submit_middleware,
+    shortcut_context_middleware,
+    user_middleware,
+)
+from dispatch.project import service as project_service
+
+
+def configure(config):
+    """Maps commands/events to their functions."""
+    pass
+
+
+def assignee_select(
+    placeholder: str = "Select Assignee",
+    initial_user: str = None,
+    action_id: str = None,
+    block_id: str = None,
+    label: str = "Assignee",
+    **kwargs,
+):
+    """Builds a assignee select block."""
+    return Input(
+        element=UsersSelect(
+            placeholder=placeholder, action_id=action_id, initial_user=initial_user
+        ),
+        block_id=block_id,
+        label=label,
+        **kwargs,
+    )
+
+
+@app.action(CaseNotificationActions.reopen, middleware=[button_context_middleware, db_middleware])
+async def reopen_button_click(
+    ack,
+    client,
+    context,
+    db_session,
+):
+    await ack()
+    case = case_service.get(db_session=db_session, case_id=context["subject"].id)
+    case.status = CaseStatus.triage
+    db_session.commit()
+
+    # update case message
+    blocks = create_case_message(case=case, channel_id=context["subject"].channel_id)
```

Comment:
```suggestion
    blocks = create_case_message(case=case, channel_id=context.channel_id)
```

context has it already

Reply:
It does, but only for buttons. Do you think using the subject for consistency across the different interactions makes sense?